config/ember-try.js

```diff
@@ -80,6 +101,14 @@ module.exports = function() {
             },
           },
         },
+        {
+          name: 'ember-4.0',
+          npm: {
+            devDependencies: {
+              'ember-source': '~4.0.1',
+            },
+          },
+        },
```

Comment:
Can we just re-enable the `ember-release` scenario instead of adding a `4.0.x`-specific scenario? That could keep us on the path of only carving out specific scenarios for supported LTS versions.